PythonTool/TagExpansion.py

```diff
@@ -171,7 +214,11 @@ def main():
     args = parse_arguments()
 
     # Expand the vector tags from the mesh file.
-    tag_expansion(args.meshfile, args.outputfile)
+    try:
+        expand_vector_tags(args.meshfile)
+    except LookupError as e:
+        print(str(e))
+        exit()
```

Comment:
This block actually is catching the exception raised on Line 177 in the `expand_vector_tags` function! So that is the message it will print out. It is organized in this way because an exception needs to be raised if there are no vector tags in the file, and therefore the function is exited at that point if the check fails and the code goes back to `main` to print the message!